src/Ankimon/functions/pokedex_functions.py

```
@@ -235,13 +255,13 @@ def get_all_pokemon_moves(pk_name, level):
     pk_name = pk_name.lower()
 
     # Retrieve the learnset for the specified Pokémon
-    pokemon_learnset = learnsets.get(pk_name, {})
+    pokemon_learnset = learnsets[pk_name]
 
     # Create a dictionary to store moves and their corresponding highest levels
     moves_at_level_and_lower = {}
 
     # Loop through the learnset dictionary
-    for move, levels in pokemon_learnset.get("learnset", {}).items():
+    for move, levels in pokemon_learnset["learnset"].items():
```

same goes here, why remove the fallback ?

---

The data files were modified so I wanted to make sure any empty fields would error (to check the new files are working correctly). In my opinion, it would be better to remove the fallbacks to prevent hiding errors that can cause further bugs later, but these changes can be reverted if you prefer the old behaviour.